doc/devel/contributing.rst

```diff
@@ -89,13 +89,13 @@ use on a preexisting environment!
 
 A new environment can be set up with ::
 
-   python3 -mvenv /path/to/devel/env
+   python3 -mvenv <file folder location>
```

Comment:
I think `/path/to/devel/env` is pretty clearly a place holder?

Reply:
I don't think so since it's windows specific & I've seen it trip up enough users that I switched to <placeholder> in all my tutorials and the like.

Reply:
What is linux-specific?  The forward slashes?

Reply:
to  be clearer, I've seen folks not recognize it as a placeholder and literally type in /path/to/dev/env

Reply:
ok?  And then it doesn't work because `/path` doesn't exist (any more than `<file folder location>` does) and they realize their mistake, right?

Reply:
As a new user contributing, I have little to no experience in setting up a new environment and took this as something I should execute verbatim. This line of text did not execute as MacOS with Catalina doesn't allow write access to that folder. This change supports a user's ability to make use of the terminal in a way that allows them to have control over where things are and take agency in that executed action.

Reply:
This is copied almost verbatim from the linked docs.  https://docs.python.org/3/library/venv.html  I think its better to stay consistent with the linked docs.

Reply:
>they realize their mistake, right?
Kinda, usually they go down a google rabbit hole of getting more confused. The path/to/folder formulation introduces more friction... especially on windows where it's path\to\folder.

And venv having confusing docs is not a reason for us to do it.

Reply:
you realize forward slashes also work on Windows, right?

Reply:
@anntzer weirdly depending on which terminal you're using...like path completion only works properly if you use slashes in one direction and not the other...and the standard documemted way is slashes in the other direction and honestly like "path/to/dev/" being similar enough to something that could work is the problem in that it makes it harder to recognize that it's a placeholder.

Reply:
I'm pretty sure path completion works on both kinds of slashes in both cmd.exe and powershell.

Reply:
> I'm pretty sure path completion works on both kinds of slashes in both cmd.exe and powershell.
I just tested it...
![ezgif-7-a965b63e1dd2](https://user-images.githubusercontent.com/1300499/92531271-f522b800-f1fb-11ea-946c-8ed4d255e7c5.gif)

Reply:
Turns out my memory failed me :/

Reply:
I agree with @jeromefv and @story645 that we should not unnecessarily assume user knowledge here. IMHO a battle of `/path/to/devel/env` vs. `<file folder location>` is somewhat futile. It's an improvement, but marginal compared to the overall issue. The whole section (probably page) assumes too much and lacks motivation, context and explanation.

IMHO it should be something like

~~~
Installing Matplotlib for development
-------------------------------------

To decouple Matplotlib development from your working python environment, we
strongly recommend using a dedicated `virtual environment`_ (or alternatively
conda environment).

The virtual environment information is stored in a folder of your choice
(called `<dev env folder>` in the following`). A new environment can be set up
with ::

   python3 -mvenv <dev env folder>

~~~

Reply:
You didn't actually follow @timhoffm 's suggestion to explain what `<file folder location>` is supposed to mean.